config/butler.php

```diff
@@ -35,6 +35,14 @@
 
     ],
 
+    'health' => [
+
+        'route' => false,
```

Comment:
This prevents butler-health from registering the default "butler-health" route, we register our own route in `routes.php`.